### ISSUE TITLE
Can't install with latest Go

### ISSUE THREAD

**Issue by AlexLast** [cloudflare/cfssl#538]
We've been experiencing issues when trying to install/update on two separate (Ubuntu 14.04) servers running different versions of Go.

We get the following output with Go 1.5.1:

# github.com/cloudflare/cfssl/scan
go/src/github.com/cloudflare/cfssl/scan/tls_handshake.go:53: undefined: tls.CipherSuites
go/src/github.com/cloudflare/cfssl/scan/tls_handshake.go:64: undefined: tls.Curves
go/src/github.com/cloudflare/cfssl/scan/tls_handshake.go:68: undefined: tls.SignatureAndHash
go/src/github.com/cloudflare/cfssl/scan/tls_handshake.go:87: undefined: tls.AllSignatureAndHashAlgorithms
go/src/github.com/cloudflare/cfssl/scan/tls_handshake.go:89: undefined: tls.SetSupportedSKXSignatureAlgorithms
go/src/github.com/cloudflare/cfssl/scan/tls_handshake.go:90: undefined: tls.ResetSupportedSKXSignatureAlgorithms
go/src/github.com/cloudflare/cfssl/scan/tls_handshake.go:93: conn.SayHello undefined (type *tls.Conn has no field or method SayHello)
go/src/github.com/cloudflare/cfssl/scan/tls_handshake.go:102: undefined: tls.Versions
go/src/github.com/cloudflare/cfssl/scan/tls_handshake.go:108: undefined: tls.CipherSuites
go/src/github.com/cloudflare/cfssl/scan/tls_handshake.go:122: undefined: tls.CipherSuites
go/src/github.com/cloudflare/cfssl/scan/tls_handshake.go:122: too many errors 

We get a similar message with 1.6 which can be provided if required.

**Comment by kisom** [comment]
This looks like the same issue reported in #523; can you try setting the environment variable GO15VENDOREXPERIMENT=1, e.g.

```
GO15VENDOREXPERIMENT=1 go get github.com/cloudflare/cfssl/cmd/...
```

**Comment by jacobhaven** [comment]
Go 1.5 should require `GO15VENDOREXPERIMENT=1` to be set, but not Go 1.6 (it's on be default)

@AlexLast if you wouldn't mind, could you try to build again with Go 1.6 and provide the error messages as well as the output of `go env`?

**Comment by AlexLast** [comment]
@jacobhaven Thanks, that fixed the issues with 1.5.

Re-tried it on go 1.6 today and it seemed to work fine, so must have been an intermittent issue :)
